hack/gosec.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+which gosec
+if [ $? -ne 0 ]; then
+	echo "Downloading gosec tool"
+	go get -u github.com/securego/gosec/v2/cmd/gosec
+fi
+
+time GO111MODULE=off gosec -conf gosec.conf.json ./...
```

Comment:
I've added this `GO111MODULE` to make it work under prow
but I dont know WHY it work (without it, we're seeing `could not determine GOARCH and Go compiler` error )
if anyone can explain...
@fromanirh @cynepco3hahue maybe?

Reply:
so with the GO111MODULE=on it works in the context of go modules and GO111MODULE=off in the context of GOPATH, so I am assuming something is configured on the PATH level that not configured on the module level(probably because of the CI)

Reply:
Did you try to run
time GOARCH=<> GOOS=<> gosec -conf gosec.conf.json ./...?